grails-app/services/de/iteratec/osm/barchart/BarchartAggregationService.groovy

```diff
@@ -62,6 +63,7 @@ class BarchartAggregationService {
                 .withJobResultDateBetween(from, to)
                 .withSelectedMeasurands(selectedMeasurands)
                 .withJobGroupIn(jobGroups)
+                .withDeviceTypes([DeviceType.TABLET])
```

Comment:
Why do you filter by tablets here? And why doesn't this break a test?

Reply:
My bad applied the git stash to the wrong tree. Was used as a debug breakpoint and should be tested in the following pr.